microservices/m2m-event-dispatcher/dev/values.yaml

```
@@ -0,0 +1,69 @@
+name: interop-be-m2m-event-dispatcher
+techStack: nodejs
+
+serviceAccount:
+  roleArn: "arn:aws:iam::505630707203:role/interop-be-m2m-event-dispatcher-dev-es1"
+
+service:
+  create: false
+
+configmap:
+  KAFKA_GROUP_ID: "{{.Values.namespace}}-m2m-event-dispatcher"
+  TOPIC_STARTING_OFFSET: "earliest"
+
+deployment:
+  flywayInitContainer:
+    create: true
+    migrationsConfigmap: "m2m-event-migrations"
+    env:
+      FLYWAY_CREATE_SCHEMAS: "true"
+      FLYWAY_PLACEHOLDER_REPLACEMENT: "true"
+      FLYWAY_URL: jdbc:postgresql://$(POSTGRES_HOST):$(POSTGRES_PORT)/$(POSTGRES_DB)
+    envFromConfigmaps:
+      FLYWAY_SCHEMAS: "{{.Values.name}}.M2M_EVENT_SCHEMA"
+      FLYWAY_PLACEHOLDERS_APPLICATIONSCHEMA: "{{.Values.name}}.M2M_EVENT_SCHEMA"
+      M2M_EVENT_DB_NAME: "common-m2m-event.M2M_EVENT_DB_NAME"
```

```suggestion
      POSTGRES_DB: "common-m2m-events-db.DB_NAME"
```
See other comment in configmap. Also see line 21 where the URL is built using these envs